assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -38,18 +38,19 @@ import { Cell, Input, TextField } from '../../material-components';
 import Button from '../Button';
 import CloseIcon from '../../../svg/close.svg';
 import { COMMA } from '../../util/key-codes';
-import VisuallyHiden from '../VisuallyHidden';
+import VisuallyHidden from '../VisuallyHidden';
 const { useSelect, useDispatch } = Data;
 
-export default function UserInputKeywords( { slug, max } ) {
+export default function UserInputKeywords( { slug, max, next, isActive } ) {
 	const keywordsContainer = useRef();
 	const [ canDeleteKeyword, setCanDeleteKeyword ] = useState( false );
 
 	const values = useSelect( ( select ) => select( CORE_USER ).getUserInputSetting( slug ) || [] );
 	const { setUserInputSetting } = useDispatch( CORE_USER );
 
 	// Add an empty string if the values array is empty.
-	if ( values.length === 0 ) {
+	const valuesLength = values.length;
```

Comment:
@asvinb is there a particular reason why we need this variable? Why not to simply use `values.length === 0 || values.length < max`?

Reply:
@eugene-manuilov Twice we need to get the length of the array instead of once.

Reply:
As far as I know, javascript doesn't recalculate length every time you read `.length` property. So there is no need to assign it to a variable because if you do it from the performance point of view, it will be even worse because javascript needs to allocate memory for your new variable and then garbage collect it.

Reply:
That's interesting. I've read mixed opinions about that but happy to go without the variable. I'll be good to compare the performance between caching the length into a variable vs not caching it.

Reply:
@asvinb see here: https://jsben.ch/5q21D - second approach (without using a variable) wins.